### DISCUSSION TITLE
游戏崩溃

### DISCUSSION THREAD

**Discussion by hejiehao** [Meloong-Git/PCL#2389] [提问] [answered]
[错误报告-2023-7-31_14.31.13.zip](https://github.com/Hex-Dragon/PCL2/files/12212106/-2023-7-31_14.31.13.zip)

**Comment by Chiloven945** [comment]
尝试使用文件补全

**Comment by WForst-Breeze** [comment, marked as answer]
#1129 已于新版本修复，如问题依旧出现请在此 Issue 下方补充信息~

**Reply by hejiehao** [reply]
我把系统UTF-8关了就能进
